### DISCUSSION TITLE
Laravel Query builder and Eloquent dont have support for MySQL INTERSECT

### DISCUSSION THREAD

**Discussion by Sourabh2111** [laravel/framework#32997] [General]
(select user_id from `skill_user_view` t1 where t1.`skill_id` = 60 and t1.`value`=1 AND short_name='yes_no_input' and `email` LIKE "%sha%")
INTERSECT
(select user_id from `skill_user_view` t2 WHERE  t2.`skill_id` = 507 and t2.`value` = 1 AND short_name='checkbox_input'  and `email` LIKE "%sha%")

How can I do this using eloquent or query builder?

**Comment by driesvints** [comment]
converted to discussion.
